XRI3TOXRI2MIGRATIONGUIDE.md

```diff
@@ -0,0 +1,86 @@
+# MRTK3 XRI2 to XRI3 migration guide
+
+## Introduction
+
+Thank you for choosing MRTK as your Mixed Reality framework.  MRTK has been upgraded to follow XRI3 patterns and consume [Unity's XR Interaction Toolkit 3+](https://docs.unity3d.com/Packages/com.unity.xr.interaction.toolkit@3.0/manual/whats-new-3.0.html) package.  The upgrade encompasses various changes from the previous MRTK version and we want to share our experience during the upgrade so that the community is aware of the changes that had to be implemented.  We also want to provide an overall roadmap for others to do their own upgrades in their forks or propietary solutions and therefore we created this guide.  We created this guide to empower you and your organization to achieve more in Mixed Reality solutions.
+
+## Original MRTK rig structure
+
+Previous to XRI3 upgrade the MRTK3 rig had a structure similar to the one shown in the next figure:
+
+<img width="522" alt="PreXRI3Structure" src="https://github.com/ms-RistoRK/MixedRealityToolkit-Unity/assets/84108471/1b104289-83a1-4b90-8d30-61bb03eae635">
```

Comment:
Won't it be an issue that all these image links are links to a personal github account, not to the main mixed reality group's account?

Reply:
Images references fixed in 26e2c268ea30496a8a6da82a5ddca61bab0f2597 , resolving.